### PR TITLE
Fix : Help interested contributors know that they can benefit from the training too!

### PR DESCRIPTION
**Pull request by oyenuga17**
**This pull request makes the following changes:**
* Fixes issue #18 

**Checklist:**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me
- [x] I certify that I ran my checklist.

Ping @Ifycode/git-github-training-resource

### CODE REVIEW

**Comment by oyenuga17** [comment]
I have made changes please review

**Comment by oyenuga17** [comment]
i have made changes please review @Ifycode

File: doc/contributors-guide/making-and-submitting-changes.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 Everyone is welcome to help improve the content of the Git and Github training resources. There are two repositories linked to the Git and Github project i.e. [learning resource repo](https://github.com/Ifycode/git-github-training-resource) and the [training repo](https://github.com/Ifycode/git-github-training). Interested contributors can follow the guides below when contributing to any of the Git and Github training repositories.
 
+{% hint style="info" %}
+ Interested contributors should note that taking the [Git and Github training](https://obiagba-mary.gitbook.io/git-and-github-training) can help you get better at contributing to the Git and Github training project (and any other Github projects they will be taking on in the future).
+{% endhint %}
```
Comment (Ifycode) [inline doc/contributors-guide/making-and-submitting-changes.md:7]: Nice job formatting as a hint @oyenuga17

Help make the following changes:
- Add a space in between the hint and the section after it. They are too close.
- Your pull request title let it begin with `Fix: `

**Review by Ifycode — CHANGES_REQUESTED** [review]

**Comment by oyenuga17** [comment]
Okay I will make the corrections

File: doc/contributors-guide/making-and-submitting-changes.md (hunk 1 above)
Comment (oyenuga17) [reply doc/contributors-guide/making-and-submitting-changes.md:7]: i have made changes at @Ifycode

Comment (Ifycode) [reply doc/contributors-guide/making-and-submitting-changes.md:7]: Thank you 👍💯

**Review by Ifycode — APPROVED** [review]

**Pull request merged**